packages/apps/human-app/frontend/src/modules/worker/utils/formatted-sign-in-error-message.ts

```diff
@@ -0,0 +1,10 @@
+import { t } from 'i18next';
+import { FetchError } from '@/api/fetcher';
+
+export function formattedSignInErrorMessage(
```

Comment:
Similar to [this comment](https://github.com/humanprotocol/human-protocol/pull/3029#discussion_r1926944226)